docs/reference/pages/model_reference.rst

```diff
@@ -356,6 +386,89 @@ Methods and properties
 
     .. automethod:: get_site_root_paths
 
+
+Locale
+======
+
+The Locale model defines the set of locales that can be used on a site.
+Each ``Locale`` record corresponds to a language code defined in the ``WAGTAIL_CONTENT_LANGUAGES`` setting.
+
+Wagtail will initially set up one ``Locale`` to act as the default language for all existing content.
+This first locale will automatically pick the value from ``WAGTAIL_CONTENT_LANGUAGES`` that most closely matches the sites primary language code defined in ``LANGUAGE_CODE``.
+If the primary language code is changed later, Wagtail will not automatically create a new ``Locale`` record or update an existing one.
+
+Before internationalisation is enabled, all pages use this primary ``Locale`` record.
+This is to satisfy the database constraints and makes it easy to switch on internationalisation later.
+
+Changing ``WAGTAIL_CONTENT_LANGUAGES``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Languages can be added or removed from ``WAGTAIL_CONTENT_LANGUAGES`` over time.
+
+Before removing an option from ``WAGTAIL_CONTENT_LANGUAGES``, it's important that the ``Locale``
+record is updated to a use a different content language or is deleted.
+Any ``Locale`` instances that have invalid content languages are automatically filtered out from all
+database queries making them unable to be edited or viewed.
+
+Methods and properties
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. class:: Locale
+    :noindex:
+
+    .. attribute:: language_code
+
+        Foo
+
+    .. automethod:: get_default
+
+    .. automethod:: get_active
+
+    .. automethod:: get_display_name
+
+
+Translatable Mixin
+==================
+
+``TranslatableMixin`` is an abstract model that can be added to any non-page Django model to make it translatable.
+Pages already have this mixin added by Wagtail, so no need to add it to them.
+
+Methods and properties
+~~~~~~~~~~~~~~~~~~~~~~
+
+The ``locale`` and ``translation_key`` fields have a unique key to prevent the object being translated into a language more than once.
```

Comment:
```suggestion
The ``locale`` and ``translation_key`` fields have a unique key constraint to prevent the object being translated into a language more than once.
```